.gitignore

```diff
@@ -8,3 +8,4 @@ target/
 /cli/pack/
 /.idea/
 /.idea_modules/
+bin/
```

Comment:
And you keep telling people that IDE specific ignores should not live in project local `.gitignore`s???